ipydatagrid/datagrid.py

```diff
@@ -525,7 +525,16 @@ def selected_cell_values(self):
         """
         List of values for all selected cells.
         """
-        return SelectionHelper(grid=self).all_values()
+        # Copy of the front-end data model
+        view_data = self.get_visible_data()
+
+        # New DataGrid instance with data from
+        # the front-end data model
+        selections_grid = DataGrid(view_data)
```

Comment:
instead of creating a new DataGrid object, can you modify `SelectionHelper`'s code to use `view_data` when needed?

Reply:
@mbektasbbg it's a significant rewrite of `SelectionHelper` which I would like to avoid. It relies on the grid's built in methods and other properties.

Reply:
I see. we should consider decoupling data model from `DataGrid` in near future and have `SelectionHelper` to work with data model directly.